hack/start-agent.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
+# arguments: TARGET, YGGDRASIL_PATH, (optional) DEVICE_WORKER_PATH
+for ARGUMENT in "$@"
+do
+   KEY=$(echo $ARGUMENT | cut -f1 -d=)
+   KEY_LENGTH=${#KEY}
+   VALUE="${ARGUMENT:$KEY_LENGTH+1}"
+   export "$KEY"="$VALUE"
+done
+
+if [[ -z $TARGET ]]; then
+    echo "Error: TARGET is required"
+    exit 1
+fi
+
+if [[ -z $YGGDRASIL_PATH ]]; then
+    echo "Error: YGGDRASIL_PATH is required"
+    exit 1
+fi
+
+if [[ $# = 3 ]]; then
+  cd $DEVICE_WORKER_PATH
+fi
+make uninstall
+make install
+
+cd $YGGDRASIL_PATH
+git checkout cdde836c519ac72302714b5f7ae90e45de7c79a7
+export PREFIX=/var/local
+make bin
+sudo chown root.root /tmp/*.pem
+
+pid_file = "/var/local/var/run/yggdrasil/workers/device-worker.pid"
+if [ -f "$pid_file" ] ; then
+    sudo rm "$pid_file"
+fi
+
+if [[ "$TARGET" == "k8s" ]]; then
+  sudo ./yggd \
+    --log-level info \
+    --protocol http \
+    --path-prefix api/flotta-management/v1 \
+    --client-id $(cat /etc/machine-id) \
+    --cert-file /tmp/cert.pem \
+    --key-file /tmp/key.pem \
+    --server flotta-operator.srv
+
+elif [[ "$TARGET" == "ocp" ]]; then
+  HTTP_SERVER=$(oc get routes flotta-operator-controller-manager -n flotta --no-headers -o=custom-columns=HOST:.spec.host)
+  sudo ./yggd \
+    --log-level info \
+    --protocol http \
+    --path-prefix api/flotta-management/v1 \
+    --client-id $(cat /etc/machine-id) \
+    --cert-file /tmp/cert.pem \
+    --key-file /tmp/key.pem \
+    --server $HOST_SERVER
```

Comment:
s/HOST_SERVER/HTTP_SERVER

Reply:
done, but I guess we'll change this again after PR #200 will be finished